sdk/communication/communication-sms/test/smsClient.spec.ts

```diff
@@ -34,70 +36,142 @@ describe("SmsClient", async () => {
     }
   });
 
-  it("sends a SMS message to multiple recipients", async () => {
-    const connectionString = env.AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING as string;
+  //helper functions
+  const expectCorrectResult = (actualSmsResult: SmsSendResult, expectedRecipient: string) => {
+    assert.equal(actualSmsResult.httpStatusCode, 202);
+    assert.equal(actualSmsResult.to, expectedRecipient);
+    assert.isString(actualSmsResult.messageId);
+    assert.isTrue(actualSmsResult.successful);
+    assert.notExists(actualSmsResult.errorMessage, "no error message for success");
+  };
+
+  const expectIncorrectResult = (
```

Comment:
nit: I would rename to `expectSuccessResult` and `expectFailureResult`. Both success and failure are "correct" in their respective outcome.